scripts/setup-env-gitbash.sh

```diff
@@ -5,13 +5,15 @@
 # Then you can use this env.sh to setup the environment.
 # (e.g. source ~/.env_uo.sh)
 
+RUBY_BASE_VERSION="3.2.0" 
+MINICONDA_VERSION="4.12.0" 
```

Comment:
We upgraded the miniconda version during the Ruby 3 upgrade: https://github.com/urbanopt/urbanopt-cli/blob/develop/lib/uo_cli.rb#L908